release/requirements_buildkite.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with python 3.7
+# This file is autogenerated by pip-compile with python 3.10
```

Comment:
we should talk about unifying our dev environment.. like for `release/` everything uses python 3.9?

where did you get 3.10?